packages/grid/_modules_/grid/components/containers/GridRoot.tsx

```diff
@@ -1,5 +1,10 @@
 import * as React from 'react';
 import clsx from 'clsx';
+import { useForkRef } from '@material-ui/core/utils';
+import { NoSsr } from '@material-ui/core';
```

Comment:
I feel like you never want to remember this one on the import path 🙃

```suggestion
import NoSsr from '@material-ui/core/NoSsr';
```